controllers/storageclaim_controller.go

```diff
@@ -202,90 +253,90 @@ func (r *StorageClassClaimReconciler) reconcilePhases() (reconcile.Result, error
 		}
 
 		// Configuration phase.
-		r.storageClassClaim.Status.Phase = v1alpha1.StorageClassClaimConfiguring
+		r.storageClaim.Status.Phase = v1alpha1.StorageClaimConfiguring
 
-		updateStorageClassClaim := false
+		updateStorageClaim := false
 		// Check if finalizers are present, if not, add them.
-		if !contains(r.storageClassClaim.GetFinalizers(), storageClassClaimFinalizer) {
-			r.log.Info("Finalizer not found for StorageClassClaim. Adding finalizer.", "StorageClassClaim", r.storageClassClaim.Name)
-			r.storageClassClaim.SetFinalizers(append(r.storageClassClaim.GetFinalizers(), storageClassClaimFinalizer))
-			updateStorageClassClaim = true
+		if !contains(r.storageClaim.GetFinalizers(), storageClaimFinalizer) {
+			r.log.Info("Finalizer not found for StorageClaim. Adding finalizer.", "StorageClaim", r.storageClaim.Name)
+			r.storageClaim.SetFinalizers(append(r.storageClaim.GetFinalizers(), storageClaimFinalizer))
+			updateStorageClaim = true
 		}
-		if utils.AddAnnotation(r.storageClassClaim, storageClientAnnotationKey, client.ObjectKeyFromObject(r.storageClient).String()) {
-			updateStorageClassClaim = true
+		if utils.AddAnnotation(r.storageClaim, storageClientAnnotationKey, client.ObjectKeyFromObject(r.storageClient).String()) {
+			updateStorageClaim = true
 		}
 
-		if updateStorageClassClaim {
-			if err := r.update(r.storageClassClaim); err != nil {
-				return reconcile.Result{}, fmt.Errorf("failed to update StorageClassClaim %q: %v", r.storageClassClaim.Name, err)
+		if updateStorageClaim {
+			if err := r.update(r.storageClaim); err != nil {
+				return reconcile.Result{}, fmt.Errorf("failed to update StorageClaim %q: %v", r.storageClaim.Name, err)
 			}
 		}
 
-		// storageClassClaimStorageType is the storage type of the StorageClassClaim
-		var storageClassClaimStorageType providerclient.StorageType
-		switch r.storageClassClaim.Spec.Type {
+		// storageClaimStorageType is the storage type of the StorageClaim
+		var storageClaimStorageType providerclient.StorageType
+		switch r.storageClaim.Spec.Type {
 		case "blockpool":
-			storageClassClaimStorageType = providerclient.StorageTypeBlockpool
+			storageClaimStorageType = providerclient.StorageTypeBlockpool
 		case "sharedfilesystem":
-			storageClassClaimStorageType = providerclient.StorageTypeSharedfilesystem
+			storageClaimStorageType = providerclient.StorageTypeSharedfilesystem
 		default:
-			return reconcile.Result{}, fmt.Errorf("unsupported storage type: %s", r.storageClassClaim.Spec.Type)
+			return reconcile.Result{}, fmt.Errorf("unsupported storage type: %s", r.storageClaim.Spec.Type)
 		}
 
-		// Call the `FulfillStorageClassClaim` service on the provider server with StorageClassClaim as a request message.
+		// Call the `FulfillStorageClassClaim` service on the provider server with StorageClaim as a request message.
 		_, err = providerClient.FulfillStorageClassClaim(
```

Comment:
What about API name changes?

Reply:
not doing now, mostly after FF as we need to take into account existing deployments as well.

Reply:
No, let me transfer to a jira task.

Reply:
done, raised a jira task.